atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/expectations/TransactionCommitLockInfo.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.impl.expectations;
+
+import org.immutables.value.Value;
+
+/**
+ * Data about the amount of commit locks requested by a transaction.
+ */
+@Value.Immutable
+public interface TransactionCommitLockInfo {
+
```

Comment:
nit: don't need the new line between the first parameter and the interface unless it has annotations and comments